scania-digital-design-system/tegel
# build: upgrade to stencil 4.22.2

## **Describe pull-request**  
Upgrading to a newer version of the Stencil core

## **Issue Linking:**  
- **No issue:** Regault optimization and upgrades of the system.

## **How to test**  
1. Clone branch
2. Run on `npm i` and `npm run build` on core
3. Run `npm run start` on root level
4. Everything should run as expected

## **Checklist before submission**
